include/OpenSpaceToolkit/Astrodynamics/Trajectory/State.hpp

```diff
@@ -81,10 +93,19 @@ class State
 
     static State Undefined();
 
+    static State FromStdVector(
+        const Instant& anInstant,
+        const std::vector<double>& aCoordinates,
+        const Shared<const Frame>& aFrameSPtr,
+        const Shared<const CoordinatesBroker> aCoordinatesBrokerSPtr
+    );
+
```

Comment:
```suggestion
```
Don't think is necessary as we are now using VectorXd in the NumericalSolver etc.

Reply:
_Tears of joy_